customer_gateway.go

```diff
@@ -10,6 +10,7 @@ type CustomerGateway struct {
 }
 
 // Create creates a new customer from the passed in customer object.
+// You can create a customer by itself, with a payment method, or with a credit card with a billing address.
 // If no Id is set, Braintree will assign one.
```

Comment:
Could you capitalize `Id`?

Reply:
@leighmcculloch if we capitalize here, should we do so in all comments ?

Reply:
Just the comments where the id being referenced is actually spelt `ID`. Unfortunately some structs use `Id` and some use `ID`.